openfisca_us/tests/baseline/benefit/SNAP/amount.yaml

```diff
@@ -0,0 +1,230 @@
+- name: 'family of 4, no member(s) with disabilities, income of 24k/year (Standard Test)'
```

Comment:
Could you just capitalize the test names like this? Then we're good to merge I think.

```suggestion
- name: Family of 4, no member(s) with disabilities, income of 24k/year (Standard Test)
```